app/src/main/java/protect/card_locker/LoyaltyCardEditActivity.java

```diff
@@ -1275,11 +1277,11 @@ private long getMinDateOfDatePicker()
         }
 
         public void onDateSet(DatePicker view, int year, int month, int day) {
-            Calendar c = Calendar.getInstance();
+            Calendar c = new GregorianCalendar();
```

Comment:
I think I saw comments about this in discord or the forum on the original project, but what happens when someone is using a diff type of calendar? (I'm pretty sure this was answered but I don't remember)
Other wise looks good!